### PR TITLE
Quickfixes for release

### PR DESCRIPTION
**Pull request by hal-ler**
To solve #522.

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -428,8 +428,8 @@ Task("OnlyPublish")
             {
                 buildIdentifier ="linux-mono";
             }
-            // No need to package for <!win7> + net451
-            else if (!runtimeShort.Contains("win7-") && framework.Equals("net451"))
+            // No need to package for <!win> + net451
+            else if (!runtimeShort.Contains("win-") && framework.Equals("net451"))
```
Comment (DustinCampbell) [inline build.cake]: Ha! Oops. :smile:

**Comment by DustinCampbell** [comment]
:+1:

**Comment by david-driscoll** [comment]
The travis change is simple, just fix https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/.travis.yml#L40

**Comment by hal-ler** [comment]
Changed ifdef DNX451 in Nuget and Roslyn.Csharp folders to ifdef NET451.
Removed ifdef DNX451 in MSBuild folder (since it only ever builds for NET451).

**Comment by hal-ler** [comment]
Mono archive renaming now only applies on Travis. This allows users to create archives for Mono on *NIX without being restricted to Ubuntu.

**Comment by hal-ler** [comment]
Travis now pushes all archives (cross-compilation already controlled by PopulateRuntime and only triggered when desired).

**Pull request merged**